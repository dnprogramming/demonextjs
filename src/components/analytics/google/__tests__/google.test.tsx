import { render } from "@testing-library/react";
import Google from "../google";
import React from "react";

describe("Google Component", () => {
    it("renders google analytics component unchanged", () => {
        const { container } = render(<Google />);
        expect(container).toMatchSnapshot();
    });
});