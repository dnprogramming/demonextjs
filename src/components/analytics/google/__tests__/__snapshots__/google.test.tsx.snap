// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Google Component renders google analytics component unchanged 1`] = `<div />`;
